core/js/typespecs/app_types.ts

```diff
@@ -331,6 +337,9 @@ const patchProjectSchema = z
 const createExperimentSchema = z
   .object({
     project_id: experimentSchema.shape.project_id,
+    project_name: experimentSchema.shape.project_name,
+    org_id: experimentSchema.shape.org_id,
+    ancestor_commits: experimentSchema.shape.ancestor_commits,
```

Comment:
We have to be careful to remove any fields we don't want to modify from the `patchExperimentSchema`.